libbeat/publisher/queue/diskqueue/serialize.go

```diff
@@ -140,8 +155,19 @@ func (d *eventDecoder) Buffer(n int) []byte {
 	}
 	return d.buf
 }
+func (d *eventDecoder) Decode() (interface{}, error) {
```

Comment:
```suggestion

func (d *eventDecoder) Decode() (interface{}, error) {
```

nit: missing space between functions.